blocks/shared-styles/_children/promo-image/index.jsx

```diff
@@ -65,6 +65,7 @@ const PromoImage = ({
       rel={newTab ? 'noreferrer noopener' : ''}
       aria-hidden="true"
       tabIndex="-1"
+      onClick={registerSuccessEvent}
```

Comment:
looks clear enough. did they give us any docs or something?

Reply:
I used this - https://corecomponents.arcpublishing.com/alc/arc-products/pagebuilder/user-docs/pagebuilder-editor-content-testing-guide/